nncf/experimental/torch/fx/quantization/quantize_model.py

```diff
@@ -68,7 +69,7 @@ def quantize_impl(
     original_graph_meta = model.meta
 
     copied_model = deepcopy(model)
-
+    shared_constant_create_transformation_builder()(copied_model)
```

Comment:
Please consider finding a way to cope with shared nodes without a model modification, model modification won't be approved

Reply:
I assumed it would be good to have a central change so that the graph is updated in one place and can be used everywhere. Otherwise in every use case we would have to search for nodes with the same target.

Reply:
We have `is_shared` method in `NNCFNode`, and we even have a ticked to support that https://github.com/openvinotoolkit/nncf/issues/2938 (duplicate from jira). Maybe it is possible to use this attribute to help with your issue?

Reply:
Oh, I will check it out. Thank you very much!

Reply:
So would you suggest I check and update this attribute for the NNCFnode in the nncf graph builder for torch fx?